pandas/tests/arrays/sparse/test_accessor.py

```diff
@@ -252,3 +252,12 @@ def test_with_column_named_sparse(self):
         # https://github.com/pandas-dev/pandas/issues/30758
         df = pd.DataFrame({"sparse": pd.arrays.SparseArray([1, 2])})
         assert isinstance(df.sparse, pd.core.arrays.sparse.accessor.SparseFrameAccessor)
+
+    def test_subclassing(self):
+        class SubclassedDataFrame(pd.DataFrame):
+            @property
+            def _constructor(self):
+                return SubclassedDataFrame
+
+        df = SubclassedDataFrame({"sparse": pd.arrays.SparseArray([1, 2])})
+        assert isinstance(df.sparse.to_dense(), SubclassedDataFrame)
```

Comment:
```suggestion
        df = tm.SubclassedDataFrame({"sparse": pd.arrays.SparseArray([1, 2])})
        assert isinstance(df.sparse.to_dense(), tm.SubclassedDataFrame)
```

Reply:
Thanks, I didn't know it's already there.